fastify4ts/.gitignore

```diff
@@ -0,0 +1,2 @@
+uploads/*
+!uploads/.gitkeep
```

Comment:
`build/` should be included here.
or, if you implement the alternative folder structure i described, `lib/`